### PR TITLE
Sleep in between API calls to respect ratelimit

### LINKED ISSUE #9: CI: github seems to have lowered the GraphQL API rate limits
Our weekly sync job has been failing for some weeks. See https://github.com/mvdan/corpus/pull/7#issuecomment-842102840.

Perhaps one fix is to add sleeps between each API request?

The requests already use a personal access token on my account, so that should be as much API access as one can get without having an enterprise account/setup.

### PR DESCRIPTION
**Pull request by khos2ow**
When calling GraphQL APIs, there's a chance of hitting rate limits [1]
or in some cases "secondary rate limits" [2]. The score of the query is
~101 and it seems sleep of 1 second in between will be enough to honor
the rate limits.

[1]: https://docs.github.com/en/graphql/overview/resource-limitations#rate-limit
[2]: https://docs.github.com/en/rest/overview/resources-in-the-rest-api#secondary-rate-limits

Fixes #9

### CODE REVIEW

**Comment by khos2ow** [comment]
@mvdan it appears adding a 1 second sleep did the trick. At first I tried with 5 seconds and then lowered it to 1 to see how it goes. The earlier result can be found [here](https://github.com/mvdan/corpus/runs/3254714250).

My original idea was to wait dynamically based on the result of the API, but in all cases the rate limit always were within range (the `cost`, `limit`, and `remaining`) and `resetAt` was always 1 hour after the start. Because of this I think this is the easiest way.

File: main.go [hunk 1]
```diff
@@ -128,6 +128,11 @@ moreLoop:
 			"repocursor":  cursor,
 			"querystring": githubv4.String(queryString),
 		}
+
+		if page > 1 {
+			time.Sleep(1 * time.Second)
+		}
```
Comment (mvdan) [inline main.go:138]: can you summarize the reason behind this sleep in a comment above this bit of code?

the commit message is great and should stay that way, but people reading the code won't be looking at the git log, so an FYI here is perhaps more important.

**Review by mvdan — APPROVED** [review]
LGTM! Just one nit.
